frontend/src/constants/API.ts

```diff
@@ -97,11 +97,15 @@ export const LOOKUP_CODE_SET = (codeSetName: string) => `/lookup/${codeSetName}`
 export const AGENCY_CODE_SET_NAME = 'Agency';
 export const ROLE_CODE_SET_NAME = 'Role';
 export const PROVINCE_CODE_SET_NAME = 'Province';
-export const AMINISTRATIVE_AREA_CODE_SET_NAME = 'AdministrativeArea';
+export const ADMINISTRATIVE_AREA_CODE_SET_NAME = 'AdministrativeArea';
```

Comment:
Seems like a reasonable reason to switch to an `enum` for these types of things.